src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -90,12 +90,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         String user = userService.getCurrentUser().getEmail();
 
-        String text = request.getParameter("text");
+        String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
         Whitelist whitelist = Whitelist.basicWithImages();
         whitelist.addTags( "h1", "h2", "h3", "h4", "h5", "h6");
         String userText = Jsoup.clean(text, whitelist);
 
-       // System.out.println(userText);
+       System.out.println(userText);
```

Comment:
This line is only for debugging.
I think I should delete this line.

Reply:
Sure.

Reply:
### Solved !!